README.md

```diff
@@ -2,70 +2,109 @@
 ```
 
-**Reference component in HTML**
+**NOTE:** Be sure to replace `@latest` in the URL with the version of the asset you want. @latest is NOT aware of any MAJOR releases, use at your own risk.
+
+### Using the custom element
+
+After the web component is imported by JavaScript or CDN, reference the component in HTML.
+
 <!-- AURO-GENERATED-CONTENT:START (CODE:src=./apiExamples/basic.html) -->
 <!-- The below code snippet is automatically added from ./apiExamples/basic.html -->
 ```html
 <auro-hyperlink>Hello World</auro-hyperlink>
 ```
 <!-- AURO-GENERATED-CONTENT:END -->
 
-## Install bundled assets from CDN
+Review the <a href="https://auro.alaskaair.com/components/auro/hyperlink/api">API documentation</a> for advanced uses and examples.
 
-In cases where the project is not able to process JS assets, there are pre-processed assets available for use.
+### Limitations and Risks
 
-### Bundle example code
+When multiple microsites are combined and result in a custom element being imported more than once, only the first instance of the imported custom element is used. All subsequent attempts to load the custom element will fail. If a microsite loaded into the document subsequently attempts to load a new version of the component with an updated API, it will fail to load. This can result in unexpected API functionality.
 
-**NOTE:** Be sure to replace `@latest` in the URL with the version of the asset you want. @latest is NOT aware of any MAJOR releases, use at your own risk.
+For a detailed explanation of this issue, see this article: https://dev.to/florianrappl/versioning-web-components-3bda
 
-```html
-<link rel="stylesheet" href="https://unpkg.com/@alaskaairux/design-tokens@latest/dist/tokens/CSSCustomProperties.css" />
-<link rel="stylesheet" href="https://unpkg.com/@alaskaairux/webcorestylesheets@latest/dist/bundled/essentials.css" />
+### Recommended Use and Version Control
 
-<script src="https://unpkg.com/@aurodesignsystem/auro-hyperlink@latest/dist/auro-hyperlink__bundled.js" type="module"></script>
+There are two important parts of every Auro Component. The <a href="https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes">class</a> and the Custom Element. The class is exported and then used as part of defining the Custom Element. When importing this component as described in the <a href="#install">install</a> section, the class is imported and the `auro-hyperlink` custom element is defined automatically.
+
+To protect from versioning conflicts with other instances of the component being loaded, it is recommended to use our `registerComponent(name)` method and pass in a unique name.
+
+```js
+import { AuroHyperlink } from './node_modules/@aurodesignsystem/auro-hyperlink';
+
+registerComponent('custom-hyperlink');
 ```
 
-### hyperlink use cases
+This will create a new custom element that you can use in your html that will function identical to `auro-hyperlink`.
```

Comment:
```suggestion
This will create a new custom element that you can use in your HTML that will function identically to the `auro-hyperlink` element.
```